lib/messages.rb

```diff
@@ -0,0 +1,57 @@
+class Messages
+
+  def pre_render_welcome
+    string = ""
+    string << "Let's play tic tac toe"
+
+    string
+  end
+
+  def pre_render_game_type
+    string = ""
+    string << "Please choose a game type:" + "\n"
+    string << "1) Human vs. Human" + "\n"
+    string << "2) Human vs. Computer" + "\n"
+    string << "3) Computer vs. Computer"
+
+    string
+  end
+
+  def pre_render_select_game_type
+    string = ""
+    string << "Please select 1, 2 or 3."
+
+    string
+  end
+
+  def pre_render_prompt_player_for_move
+    string = ""
+    string << "Please make your move"
+
+    string
+  end
+
+  def pre_render_prompt_select_space
+    string = ""
+    string << "Please select a valid space on the board."
```

Comment:
Why do you go through the trouble of creating a string, altering, then returning it when you could simply return a string.